include/bbp/sonata/config.h

```diff
@@ -645,6 +646,30 @@ class SONATA_API SimulationConfig
         bool representsPhysicalElectrode = false;
     };
 
+    struct EField {
+        /// Peak amplitude of the sinusoid in the x-direction, in V/m
+        double ex{};
+        /// Peak amplitude of the sinusoid in the y-direction, in V/m
+        double ey{};
+        /// Peak amplitude of the sinusoid in the z-direction, in V/m
+        double ez{};
+        /// Frequency of the sinusoid, in Hz. Must be non-negative. If not provided, assumed to be 0
+        double frequency{};
```

Comment:
do we want to consider dissipation too? In this case we want a complex value for the frequency